### PR TITLE
docs(firebase_auth): fixed Facebook Authentication example

### PR DESCRIPTION
**Pull request by darwin-morocho**
## Description

I have been updated the example code using the flutter_facebook_auth plugin because the LoginResult class was removed in the version ^1.0.0

## Related Issues
No Related Issues

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [x] Yes, this is a breaking change ([`CHANGELOG.md`](https://pub.dev/packages/flutter_facebook_auth/changelog#100)).
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: docs/auth/social.mdx [hunk 1]
```diff
@@ -130,22 +130,25 @@ dependencies:
...
-  // Trigger the sign-in flow
-  final LoginResult result = await FacebookAuth.instance.login();
-
-  // Create a credential from the access token
-  final FacebookAuthCredential facebookAuthCredential =
-    FacebookAuthProvider.credential(result.accessToken.token);
-
-  // Once signed in, return the UserCredential
-  return await FirebaseAuth.instance.signInWithCredential(facebookAuthCredential);
+  try {
```
Comment (rrousselGit) [inline docs/auth/social.mdx:139]: Why the try/on/finally here?

Comment (darwin-morocho) [reply docs/auth/social.mdx:139]: Hi @rrousselGit  now the `FacebookAuth.instance.login()` method throw a FacebookAuthException when the facebook login request failed. 

[here](https://pub.dev/packages/flutter_facebook_auth#methods)  one example

Comment (rrousselGit) [reply docs/auth/social.mdx:139]: Yes but why should the snippet catch the error?

**Comment by rrousselGit** [comment]
Looking at the most up-to-date documentation of `facebook_auth`, it appears that `login` correctly returns a `LoginResult`, not an `AccessToken`:

https://pub.dev/documentation/flutter_facebook_auth/latest/flutter_facebook_auth/FacebookAuth/login.html

So I'm going to close this

**Pull request closed without merging**